src/models/exercise.model.ts

```diff
@@ -0,0 +1,6 @@
+import { Set } from './set.model'
+
+export interface Exercise {
```

Comment:
There's already an interface exported called `Exercise` (you'll need to merge this branch with `dev` to see it). I think it's going to get super confusing. 

I think we need to tweak the naming of these interfaces, as well as Session and Set. 
`ExerciseSession`, `ExerciseSet`?

and maybe rename this to `ExerciseActivity`? 
You could also type the `exerciseName` property to use the `ExerciseEnums | string`